jobs/pcap-api/templates/pcap-api.yml.erb

```diff
@@ -10,15 +10,20 @@ config = {
   "concurrent_captures" => p("pcap-api.concurrent_captures"),
   "listen" => {
     "port" => p("pcap-api.listen.port"),
-    "tls" => {
-      "certificate"=> "/var/vcap/jobs/pcap-api/config/certs/pcap-api.crt",
-      "private_key" => "/var/vcap/jobs/pcap-api/config/certs/pcap-api.key",
-      "ca" => "/var/vcap/jobs/pcap-api/config/certs/pcap-api-ca.crt",
-    },
   },
   "cli_download_root" => p("pcap-api.cli_download_root")
 }
 
+if_p("pcap-api.listen.tls.certificate", "pcap-api.listen.tls.private_key", "pcap-api.listen.tls.ca") do
```

Comment:
As much as I hate the `enabled` property for such things, after careful consideration I suggest we do use it here for a few reasons:
- This will fail in unexpected ways, e.g. you don't provide a CA and get no TLS at all without any error.
- We cannot check if the parent node `pcap-api.listen.tls` exists (thanks to bosh).
- With the `enabled` property we will explicitly fail if it is `true` but one of the required fields is missing.

WDYT?

(sorry that I suggested the opposite yesterday)

Reply:
I agree, but want to get more feedback before implementing.

Reply:
See https://github.com/cloudfoundry/pcap-release/pull/23/commits/f4553fbb7302ad917ff8e4b8413eb93c7c830fe6